In [1]:
import pandas as pd

In [3]:
wine_df = pd.read_csv('/content/wines_SPA.csv')
meta_df = pd.read_csv('/content/Metabolic  Syndrome.csv')

# Data set 1: Wines

In [4]:
wine_df.head()

winery           wine  year  rating  num_reviews country  \
0  Teso La Monja          Tinto  2013     4.9           58  Espana   
1         Artadi  Vina El Pison  2018     4.9           31  Espana   
2   Vega Sicilia          Unico  2009     4.8         1793  Espana   
3   Vega Sicilia          Unico  1999     4.8         1705  Espana   
4   Vega Sicilia          Unico  1996     4.8         1309  Espana   

             region   price                  type  body  acidity  
0              Toro  995.00              Toro Red   5.0      3.0  
1    Vino de Espana  313.50           Tempranillo   4.0      2.0  
2  Ribera del Duero  324.95  Ribera Del Duero Red   5.0      3.0  
3  Ribera del Duero  692.96  Ribera Del Duero Red   5.0      3.0  
4  Ribera del Duero  778.06  Ribera Del Duero Red   5.0      3.0

The dataset is related to spanish wine partivullarly the red variant. This data set contains 11 features. 

The traget i have chosen is to predict the quality of the wine.

In this dataset one of the rows reprasent a prouduct Rating from other people, acidity, or the Wine itself.

This dataset and the traget i have chosen made this dataset a classification problem because I am predicting the quality of wine

This data set contains 11 features

winery,	wine,	year,	rating,	num_reviews,	country	region,	price,	type,	body,	acidity

In [9]:
wine_df.shape

(7500, 11)

There are 7,500 rows in this data set

There is some missing values is this dataset. I will need to clean the data set

# Data Set 2: Metabolic Syndrome Prediciton

In [5]:
meta_df.head()

seqn  Age     Sex  Marital  Income   Race  WaistCirc   BMI  Albuminuria  \
0  62161   22    Male   Single  8200.0  White       81.0  23.3            0   
1  62164   44  Female  Married  4500.0  White       80.1  23.2            0   
2  62169   21    Male   Single   800.0  Asian       69.6  20.1            0   
3  62172   43  Female   Single  2000.0  Black      120.4  33.3            0   
4  62177   51    Male  Married     NaN  Asian       81.1  20.1            0   

   UrAlbCr  UricAcid  BloodGlucose  HDL  Triglycerides MetabolicSyndrome  
0     3.88       4.9            92   41             84         No MetSyn  
1     8.55       4.5            82   28             56         No MetSyn  
2     5.07       5.4           107   43             78         No MetSyn  
3     5.22       5.0           104   73            141         No MetSyn  
4     8.13       5.0            95   43            126         No MetSyn

This dataset is to predict metabolic syndrome in 

The target for this dataset is to predict metabolic syndrome based on the common risk factors.

A row in this dataset reprasents a person/patiant and all their information.

This is a classification problem cause were predicting if yes or no the person has a Metabolic Syndrome.

There are 15 features in this dataset

seqn,
age,
sex,
marital,
income,
race,
waistcirc,
bmi,
albuminuria,
uralbcr,
uricacid,
bloodglucose,
hdl,
triglycerides,
metabolicsyndrome,


In [10]:
meta_df.shape

(2401, 15)

There are 2,401 rows in this dataset

There is some missing values and NaN in the dataset. I will also have to drop some columns and clean it.